qiskit/algorithms/linear_solvers/observables/expected_value.py

```diff
@@ -0,0 +1,57 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The expected value of an operator applied to the vector solution to the linear systems."""
+from typing import Union, Optional
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.providers import BaseBackend, Backend
+from qiskit.aqua import QuantumInstance
+from qiskit.aqua.operators.operator_base import OperatorBase
+from .linear_system_observable import LinearSystemObservable
+
+
+class ExpectedValue(LinearSystemObservable):
+    """A class for the expected value of an operator applied to the vector solution
+    to the linear systems."""
+
+    def __init__(self, op: OperatorBase, tolerance: Optional[float] = None,
+                 quantum_instance: Optional[Union[QuantumInstance, BaseBackend, Backend]] = None) \
+            -> None:
+        """
+        Args:
+            op: The operator to compute the expected value.
+            tolerance: error tolerance.
+                Defaults to ``1e-2``.
+            quantum_instance: Quantum Instance or Backend
+        """
+        super().__init__(tolerance, quantum_instance)
+
+        self._op = op
+        self._tolerance = tolerance if tolerance is not None else 1e-2
```

Comment:
Can we use default vals in the signature instead?

Reply:
this file will be deleted